module2-database-technologies/README.md

```diff
@@ -1 +1,372 @@
-# curriculum-backend-readings
\ No newline at end of file
+# Module 2: Databases
+
+## Introduction to Databases
+In this module, you'll learn about **databases**. Database is a general term
+that refers to the data stored in a structured format. We use databases when we
+want to persist data for future use. For example, in a web application, when a
+user writes something in a text box, this data will be lost when the page is
+refreshed -- unless this data is persisted in a database.
+
+Databases are a ubiquitous and foundational part of backend technology.
```

Comment:
This should also include "When are databases used? (As opposed to say, local storage. Or cookies.)". Checking from the module outline.